### DISCUSSION TITLE
How to Insert Parquet Data into ClickHouse with clickhouse-js

### DISCUSSION THREAD

**Discussion by YCUBEANR** [ClickHouse/clickhouse-js#206] [Q&A] [answered]
Is it possible to pass Parquet files with clickhouse-js

```
 .insert({
    table: `${dbConfig.database}.${dbConfig.table}`,
    values: fs.createReadStream(path),
    format: 'Parquet',
  })
```
Getting errors but not sure this is even in scope

**Comment by slvrtrn** [comment, marked as answer]
Sorry for the late reply - just noticed that I had no subscription for discussions (and, consequently, no notifications for that).
Should be possible in 0.2.6 (once this is merged https://github.com/ClickHouse/clickhouse-js/pull/208)
